src/browser/page.zig

```diff
@@ -461,9 +474,9 @@ pub const Page = struct {
         };
 
         var script_source: ?[]const u8 = null;
+        defer self.current_script = null;
```

Comment:
moving this outside the `if`  block, so that `self.current_script` is set when `script.eval` is called is the only meaningful change in this PR.